testgrid/config.yaml

```diff
@@ -4449,6 +4449,8 @@ dashboards:
     test_group_name: ci-kubernetes-e2e-gci-gce-autoscaling-migs
   - name: gci-gke-autoscaling
     test_group_name: ci-kubernetes-e2e-gci-gke-autoscaling
+  - name: gci-gke-autoscaling-gpu
+    test_group_name: ci-kubernetes-e2e-gci-gke-autoscaling-gpu
```

Comment:
you need to define this test_group at the top, you can search for all references for `ci-kubernetes-e2e-gci-gce-autoscaling-migs` for example